cas-server-core/src/main/java/org/jasig/cas/CasVersion.java

```diff
@@ -43,4 +47,27 @@ private CasVersion() {
     public static String getVersion() {
         return CasVersion.class.getPackage().getImplementationVersion();
     }
+
+    /**
+     * Gets last modified date/time for the module.
+     * @return the date/time
+     */
+    public static DateTime getDateTime() {
```

Comment:
It's the date time of the `CasVersion` class: so it can be seen as the date time of the last installation / update of the CAS server, cannot it?

I'm a bit skeptical of its use.